docscan/sandbox/request/check/supplementary_document_text_data_check.go

```diff
@@ -0,0 +1,75 @@
+package check
+
+import (
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/check/report"
+	"github.com/getyoti/yoti-go-sdk/v3/docscan/sandbox/request/filter"
+)
+
+// DocumentTextDataCheck Represents a document text data check
+type SupplementaryDocumentTextDataCheck struct {
+	Result SupplementaryDocumentTextDataCheckResult `json:"result"`
+	*documentCheck
+}
+
+// SupplementaryDocumentTextDataCheckBuilder builds a SupplementaryDocumentTextDataCheck
+type SupplementaryDocumentTextDataCheckBuilder struct {
+	documentCheckBuilder
+	documentFields map[string]interface{}
+}
+
+// SupplementaryDocumentTextDataCheckResult represent a document text data check result
+type SupplementaryDocumentTextDataCheckResult struct {
+	checkResult
```

Comment:
Was going to ask if we should be using a pointer to `*checkResult` here, but looks like we haven't done that in the other checks, so probably best to be consistent